src/table/pagination-table/components/head/styles.ts

```diff
@@ -4,14 +4,24 @@ import { COMMON_CELL_STYLING } from "../../../styling-defaults";
 
 // eslint-disable-next-line import/prefer-default-export
 export const StyledHeadCell = styled(TableCell, {
-  shouldForwardProp: (prop: string) => !["headerStyle", "isNewHeadCellMenuEnabled"].includes(prop),
-})(({ headerStyle, isNewHeadCellMenuEnabled }) => ({
+  shouldForwardProp: (prop: string) =>
+    prop !== "headerStyle" &&
+    prop !== "isNewHeadCellMenuEnabled" &&
+    prop !== "interactions" &&
+    prop !== "hoverBackground" &&
+    prop !== "background",
```

Comment:
Looks like `background` is only used with the feature flag. Was it not used before? So without the flag it should have not background?

Reply:
headerStyle has background property on it: https://github.com/qlik-oss/sn-table/blob/d1cf06c376ded408c730b2a05ab99fe4091dc334/src/table/utils/styling-utils.ts#L130

so the active and hover background is derived from that background color ([in styling utils here](https://github.com/qlik-oss/sn-table/blob/44ab786649a287bd3874108a1637dc663b39f9ca/src/table/utils/styling-utils.ts#L145-L156))